sdk/rust/nym-sdk/examples/tcp_proxy.rs

```diff
@@ -0,0 +1,96 @@
+use nym_sdk::tcp_proxy;
+use rand::Rng;
+use tokio::io::AsyncWriteExt;
+use tokio::net::{TcpListener, TcpStream};
+use tokio::task;
+use tokio_stream::StreamExt;
+use tokio_util::codec;
+
+// This is a basic example which opens a single TCP connection and streams between a client and
+// server, so only uses a single session under the hood. See <MORE ELABORATE EXAMPLE> for use of
+// multiple sessions/streams.
+#[tokio::main]
+async fn main() {
```

Comment:
I'd use `anyhow` here as well
```suggestion
async fn main() -> anyhow::Result<()> {
```